Beis.LearningPlatform.Web/Controllers/SkillsThreeController.cs

```diff
@@ -44,6 +44,8 @@ public async Task<IActionResult> Result(DiagnosticToolForm model)
             var response = await _controllerHelper.ProcessResults(model, GetFormType());
             if (response.Result && response.Payload)
             {
+                string completedSessionKey = $"{model.userTypeActionPlanSection}__CompletedLink";
+                HttpContext.Session.SetString(completedSessionKey, "true");
```

Comment:
Should be able to used HttpContextAccessor similar to _httpContextAccessor.HttpContext.Request

Reply:
Changed as above